src/shellbar.scss

```diff
@@ -14,50 +14,56 @@ $block: #{$fd-namespace}-shellbar;
 
   // Paddings
-  $fd-shellbar-outer-spacing-lr-large: map-get($fd-spacing, "m") !default;
-  $fd-shellbar-outer-spacing-lr-medium: map-get($fd-spacing, "s") !default;
-  $fd-shellbar-outer-spacing-lr-small: map-get($fd-spacing, "xxs") !default;
-  $fd-shellbar-outer-spacing-tb: map-get($fd-spacing, "base") !default;
-  $fd-shellbar-item-spacing: map-get($fd-spacing, "xs") !default;
+  $fd-shellbar-item-spacing: 0.5rem !default;
 
   // Heights
-  $fd-shellbar-height: 48px !default;
+  $fd-shellbar-height: 2.75rem !default;
   $fd-shellbar-copilot-width: 30px !default;
 
   // Colors
-  $fd-shellbar-color: fd-color("text", 5) !default; // White
-  $fd-shellbar-background-color: fd-color("shell", 1) !default;
+  $fd-shellbar-color: var(--sapShell_TextColor) !default; // White
+  $fd-shellbar-background-color: transparent !default;
+  $fd-field-border-color: var(--sapField_BorderColor) !default;
+  $fd-shell-active-text-color: var(--sapShell_Active_TextColor) !default;
+
+  // Background
+  $fd-shellbar-background: var(--sapShellColor) !default;
+  $fd-shell-hover-background: var(--sapShell_Hover_Background) !default;
+  $fd-shell-active-background: var(--sapShell_Active_Background) !default;
+  $fd-content-badge-background: var(--sapContent_BadgeBackground) !default;
+
+  // Box shadow
+  $fd-shellbar-box-shadow: inset 0 -0.0625rem 0 0 var(--sapShellBorderColor) !default;
 
   @include fd-reset();
-  @include fd-var-color("background-color", $fd-shellbar-background-color, --fd-color-shell-1);
 
+  background-color: $fd-shellbar-background;
+  box-shadow: $fd-shellbar-box-shadow;
   height: $fd-shellbar-height;
   display: flex;
   align-items: center;
-
-  @include fd-var-color("color", $fd-shellbar-color, --fd-color-text-5);
-
+  color: $fd-shellbar-color;
   position: relative;
   justify-content: space-between;
 
   @include fd-screen(s) {
-    padding: $fd-shellbar-outer-spacing-tb $fd-shellbar-outer-spacing-lr-small;
+    padding: 0 1rem;
   }
 
   @include fd-screen(m) {
-    padding: $fd-shellbar-outer-spacing-tb $fd-shellbar-outer-spacing-lr-medium;
+    padding: 0 1rem;
   }
 
   @include fd-screen(l) {
-    padding: $fd-shellbar-outer-spacing-tb $fd-shellbar-outer-spacing-lr-large;
+    padding: 0 1rem;
   }
 
   @include fd-screen(xl) {
-    padding: $fd-shellbar-outer-spacing-tb $fd-shellbar-outer-spacing-lr-large;
+    padding: 0 1rem;
   }
 
   @include fd-screen(xxl) {
-    padding: $fd-shellbar-outer-spacing-tb $fd-shellbar-outer-spacing-lr-large;
+    padding: 0 1rem;
   }
```

Comment:
The padding is the same for all screens. Can we remove these lines and just add `padding: 0 1 rem;` to the `fd-shellbar` class?